Update Pico CSS to v2.1.1

paskal opened the pull request:
## Summary
- Updated Pico CSS framework from the previously included version to v2.1.1
- Updated import reference in jane.scss to use new 'content/misc' module name instead of 'content/miscs'

paskal commented:
I haven't checked if https://github.com/picocss/pico/commit/008e05241952535a7d069256c17d8dcf81c852dc is still actual.

xianmin commented:
Our pico is managed using subtree, and I'll check more about this pico version later.

paskal committed:
Merge commit '6f46f319f4dee3083d5e9fbf57677f4f8a1f4618' into update-pico-css-to-v2.1.1

paskal committed:
Squashed 'assets/sass/pico/' changes from 008e0524..1039a478

1039a478 Merge pull request #662 from picocss/dev
ceb77006 chore: bump version to v2.1.1
76440e6d chore: build
651e89ea Revert "fix: don't count hidden rows in table stripes"
2e66cc8b Merge pull request #658 from picocss/dev
79d239a6 chore: bump version to v2.1.0
9165905b chore: lint
bad6cafd chore: upgrade dependencies
537e3aed refactor: replace deprecated global functions to fix Sass warnings
0a77e6f3 refactor: replace deprecated color functions to fix Sass warnings
64bfd3d7 chore: remove unnecessary Node.js version files
b321a0b2 chore: remove unnecessary node version constraint
1e8fd28a chore: update Twitter url to X
0674def2 chore: upgrade dependencies
fece6794 chore: build
c38b5180 Merge pull request #587 from panda-madness/patch-1
4a0fd59d chore: remove silence SASS deprecation mixed-decls
4e474076 chore: build
1d1006e8 Merge pull request #596 from nstringham/mixed-sass-declarations
e66b84b5 Merge branch 'dev' into pr/596
27e7a047 Merge branch 'dev' of https://github.com/picocss/pico
69f3dcf4 chore: build
96ad16ee Merge pull request #615 from mwargan/patch-1
d44f6d37 chore: build
bd06c1cb Merge pull request #625 from coezbek/main
5c493e64 chore: build
4b57da2f Merge pull request #585 from feored/fix-striped-table
6ea95309 chore: upgrade dependencies
f096142b Merge pull request #603 from picocss/dependabot/npm_and_yarn/micromatch-4.0.8
cb1e8ce3 Merge pull request #636 from picocss/dependabot/npm_and_yarn/cross-spawn-6.0.6
b5dd2c01 chore: build
559213c1 Merge pull request #644 from ISSOtm/patch-1
ae9f842e chore: build
d2338228 Merge pull request #641 from rcoup/web-components-host
4311b36a Merge pull request #538 from Rotzbua/fix_slash_void_element
bbe3cdd0 Merge pull request #613 from preist/fix-typos
8b6fe747 chore: build
4c3743e6 Merge pull request #528 from Zurga/fix/modal_with_card
9b582214 chore: upgrade dependencies
26e7cf9f Style the `samp` element like `code`
02c9fb69 Support web components/shadow DOM.
77c5c1d3 chore(deps): bump cross-spawn from 6.0.5 to 6.0.6
b3a731bb Update _loading.scss not to affect form elements itself
6f3abc2c Fix details inside drop-down being rendered as dropdown
dc7f6d0a Fix several typos throughout the project
3e72ebfd chore(deps): bump micromatch from 4.0.5 to 4.0.8
f5162073 chore: build
c3bcecbe refactor: move nested declarations after non-nested declarations
6fabc27b Update _tooltip.scss
72e46994 fix: don't count hidden rows in table stripes
f15bde60 fix(docs): remove trailing slash from html5 void elements
0810f1e7 fix: Card in modal does not use modal styling
REVERT: 008e0524 fix "Can't find stylesheet to import" in hugo dart-sass

git-subtree-dir: assets/sass/pico
git-subtree-split: 1039a4788d6abc368d5485ae6bac84a8f0e3096f

paskal committed:
Update Pico CSS to v2.1.1

- Updated Pico CSS from v2.0.6 to v2.1.1 using git subtree
- Updated jane.scss import to use 'content/misc' instead of 'content/miscs' to match Pico CSS file rename

paskal commented:
Redone it using `git subtree pull --prefix=assets/sass/pico https://github.com/picocss/pico.git v2.1.1 --squash`.
